### DISCUSSION TITLE
Copilot is Crashing

### DISCUSSION THREAD

**Discussion by Finnwww** [drknzz/GitHub-Achievements#229] [Q&A] [answered]
Select Topic Area
Product Feedback

Body
I paid for a subscription to copilot this month but am having problems with Copilot crashing or stopping. I am running Linux Mint 21.1 with Dotnet 6 and 7, VS Code, and C#. The problem happens regularly under the following conditions:

I have a solution with C# files.
If I open VS Code with this solution the Copilot works by suggesting code.

but *
If I switch to another C# source code file inside the solution then back to the original file copilot either crashes or stops working altogether.

The only way to get Copilot working again is to exit and restart VS Code.

What is going on and what steps can I take to gather more information about this problem in order to forward it to the developers in charge of Copilot and VS Code integration?

**Comment by kunyiwang** [comment, marked as answer]
Check the VS Code and Copilot extension logs. You can find the log files in the following location:
Linux: ~/.vscode/logs/

Check for any other extensions you have installed that may be causing conflicts with Copilot. Try disabling other extensions temporarily to see if the issue persists.

Other than that great feedback. Hopefully this can be patched.

**Comment by bettyc0der** [comment]
When I have issues such as these i usually will uninstall and reinstall copilot
